dimod/reference/samplers/exact_solver.py

```diff
@@ -159,6 +159,53 @@ def sample_poly(self, polynomial, **kwargs):
         """
         return ExactSolver().sample(polynomial, **kwargs)
 
+class ExactDQMSolver(Sampler):
+    """A simple exact solver for testing and debugging code using your local CPU.
+
+    Notes:
+        This solver becomes slow for problems with 18 or more
+        variables, worse for increasingly many cases per variable.
+
+    """
+    properties = None
+    parameters = None
+
+    def __init__(self):
+        self.properties = {}
+        self.parameters = {}
+
+    def sample_ising(self, h, J, **parameters):
+        return ExactSolver().sample_ising(h, J, **parameters)
+
+    def sample_qubo(self, Q, **parameters):
+        return ExactSolver().sample_qubo(Q, **parameters)
```

Comment:
Agreed. I would only keep `.sample_dqm` to be consistent with the DQM solver.

Reply:
I believe there is an issue of instantiating the sampler using the Sampler base class without sample_ising and sample_qubo methods; otherwise I agree with you both.

Reply:
Do you need to subclass from `Sampler`?  It looks like `LeapHybridDQMSampler` does not.

Reply:
My bad, you only need one of `sample`, `sample_ising`, `sample_qubo`. I can remove the latter two and still make use of `Sampler`.

Reply:
`Sampler` is only defined for BQM samplers (if we had known we were going to support other types we would have called it `BQMSampler` but :shrug: ). Agree with John, it just shouldn't subclass.